CHANGELOG.md

```diff
@@ -26,6 +26,14 @@ Classify the change according to the following categories:
     ##### Removed
     ### Patches
 
+## Develop 03/08/2022
+### Minor Updates
+##### Changed
+- Updated the following default values:
```

Comment:
@bpulluta it would just be good to specify here that this was in the job/ app (v3), to distinguish between default updates made to different versions.